.gitignore

```diff
@@ -1,4 +1,5 @@
 Assemblies/CombatExtended.dll
+Assemblies/0CombatExtendedLoader.dll
```

Comment:
Unfortunately, we need the loader embedded in the repository, since it's what pops up the "you're running uncompiled dev" warning.  Other than on major RW version updates, it doesn't need to change, so we "live with it".  If you have a better solution, great, but otherwise we can't blacklist this (and just have to reject PRs which rebuild it without real changes).